projects/js-packages/components/components/admin-page/footer.jsx

```diff
@@ -22,13 +25,13 @@ const AdminPageFooter = props => {
 
 	return (
 		<div className="jp-admin-page-section">
-			<div className="jp-wrap">
-				<div class="jp-row">
-					<div class="lg-col-span-12 md-col-span-8 sm-col-span-4">
+			<Wrap>
+				<Row>
+					<Grid lg="12" md="8" sm="4">
```

Comment:
I would say that's better to receive a number than string, since we could use `PropTypes` to warn some different value.

Reply:
Done that. Even though propTypes only warns on development mode

Reply:
@leogermani Think is missing to update here to use `number` no?